samples/KafkaFlow.Sample.Dashboard/README.md

```diff
@@ -19,6 +19,14 @@ Position the terminal in that folder and run the following command.
 docker-compose up -d
 ```
 
+### Build the dashboard UI
+
+Using your terminal of choice, navigate to `kafkaflow\src\KafkaFlow.Admin.Dashboard\ClientApp` folder and run the following command:
+
+```bash
```

Comment:
The resource files aren't being bundled on the Dashboard package:
https://nuget.info/packages/KafkaFlow.Admin.Dashboard/3.0.4
How is this working for someone who installs the package?

Reply:
From what we have discussed offline, I think this is also bundled on the .dll